tests/main/snapctl-from-snap/task.yaml

```diff
@@ -39,11 +41,11 @@ execute: |
     check_cookie_exists $COOKIE_FILE
 
     echo "Verify that snapctl get can be executed by the app and shows the value set by configure hook"
-    /snap/bin/snapctl-from-snap.snapctl-get foo | MATCH bar
+    $SNAPMOUNTDIR/bin/snapctl-from-snap.snapctl-get foo | MATCH bar
 
     echo "Verify that snapctl set can modify configuration values"
-    /snap/bin/snapctl-from-snap.snapctl-set foo=123
-    /snap/bin/snapctl-from-snap.snapctl-get foo | MATCH 123
+    $SNAPMOUNTDIR/bin/snapctl-from-snap.snapctl-set foo=123
+    $SNAPMOUNTDIR/bin/snapctl-from-snap.snapctl-get foo | MATCH 123
```

Comment:
... and no quotings here, which is fine. Just raising here to say that a _lot_ of our content isn't quoted, and this isn't a real problem we have today.